### PR TITLE
WIP: Alert Streams and Filters #1

### PR DESCRIPTION
**Pull request by dmitryduev**
This PR introduces the infrastructure for managing alert streams and filters in Fritz.

Concretely:

- `extensions/skyportal/configs/db_fritz.yaml` is a first draft of the the seed config file to be used when initializing Fritz
  - Alert streams are defined in the config
- Fritz-specific K & SP data model adjustments + API endpoints to deal with alert streams and filters. The respective concerns of K and SP are separated as much as possible.
- Reworked and expanded Group management page
- Filter management page
- A couple MUI-re-skinned components here and there :)

streams01

The scree capture ⬆ is slightly outdated -- now all successful actions are accompanied by notifications.

### CODE REVIEW

**Comment by dmitryduev** [comment]
TODO: 
- changes in `data_loader.py` + `db_firtz.yaml` should take into account @stefanv's latest PR
- several test are missing
- test filter button doesn't do anything currently

But otherwise should be ready to be looked at, @stefanv @dannygoldstein -- mostly to check if there's anything glaring that needs to be fixed straight away.

**Comment by profjsb** [comment]
Small UI thing: I think fields that say "email" or "email addresses" should be changed to "user names" or "usernames". We've been using username == email address in Skyportal as the default but this is not enforced anywhere and, actually, I prefer usernames to be potentially distinct from emails.

File: extensions/skyportal/static/js/components/Group.jsx [hunk 1]
```diff
@@ -0,0 +1,568 @@
+import React, {useEffect, useState} from 'react';
```
Comment (acrellin) [inline extensions/skyportal/static/js/components/Group.jsx:1]: @dmitryduev could these extensions/duplications be obviated if we simply added a few features/changes on the SP side?

Comment (dmitryduev) [reply extensions/skyportal/static/js/components/Group.jsx:1]: that's the hope :) I'm happy to submit the corresponding PRs on the SP side once we converge on what could be "transferred"

**Comment by dmitryduev** [comment]
Closing in favor of #21

**Pull request closed without merging**